permissions/plugin-coverage.yml

```
@@ -0,0 +1,9 @@
+---
+name: "coverage"
+github: "jenkinsci/coverage-plugin"
+issues:
+  - jira: 'TODO'  # coverage-plugin
```

```suggestion
  - jira: '29223'  # coverage-plugin
```

As per https://issues.jenkins.io/rest/api/2/component/29223